worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -2176,64 +2176,70 @@ public int hollowOutRegion(Region region, int thickness, Pattern pattern) throws
     }
 
     /**
-     * Draws a line (out of blocks) between two vectors.
+     * Draws a line (out of blocks) between two or more vectors.
      *
      * @param pattern The block pattern used to draw the line.
-     * @param pos1 One of the points that define the line.
-     * @param pos2 The other point that defines the line.
+     * @param vectors the list of vectors to draw the line between
      * @param radius The radius (thickness) of the line.
      * @param filled If false, only a shell will be generated.
      *
      * @return number of blocks affected
      * @throws MaxChangedBlocksException thrown if too many blocks are changed
      */
-    public int drawLine(Pattern pattern, BlockVector3 pos1, BlockVector3 pos2, double radius, boolean filled)
+    public int drawLine(Pattern pattern, List<BlockVector3> vectors, double radius, boolean filled)
```

Comment:
this is a breaking signature change, please leave an overload

Reply:
Oh yeah, not sure why that didn't cross my mind. Will fix.

Reply:
Do you think a
```
@see #drawLine(Pattern, List, double, boolean) 
```
doc addition to the overload for the old signature would be good or shall I rather leave it out?

Reply:
I think I personally tend to copy the whole thing (and change the individual params), you can probably check some other overloaded methods here in EditSession. `@see` is probably ok too, definitely wouldn't leave it blank.

Reply:
IMO it would be a good idea to deprecate the old version too, just to keep the API small in the future. Not sure how wiz feels about that.

Reply:
while I am not against deprecation here, we do have a lot of convenience overloads already (eg point+apothem vs region args). would we want to deprecate those too?